src/subdomains/generic/user/models/account-merge/account-merge.service.ts

```diff
@@ -84,6 +87,6 @@ export class AccountMergeService {
   }
 
   private buildConfirmationUrl(code: string): string {
-    return `${Config.url()}/auth/mail/confirm?code=${code}`;
+    return `${Config.frontend.services}/kyc?merge-code=${code}`;
   }
```

Comment:
Are you sure you want to do it on the KYC screen? Maybe a separate screen is easier?

Reply:
Adapt Mail with new link